src/components/nav/NavUser.tsx

```diff
@@ -31,14 +34,19 @@ export function NavUser({
             <SidebarMenuButton
               size="lg"
               className="data-[state=open]:bg-sidebar-accent data-[state=open]:text-sidebar-accent-foreground"
-              tooltip={user.name}
+              tooltip={user.name || "Unknown User"}
             >
               <Avatar className="h-8 w-8 rounded-lg">
-                <AvatarImage src={user.avatar} alt={user.name} />
-                <AvatarFallback className="rounded-lg">{user.name}</AvatarFallback>
+                <div
+                  className="w-full h-full flex items-center justify-center text-white font-semibold text-sm"
```

Comment:
You may use the custom background color directly with the `bg-` class utiltity to avoid creating extra variables and passing it down through properties, for example: `bg-[#f59e0b]`

```suggestion
                  className="w-full h-full flex items-center justify-center text-white font-semibold text-sm bg-[#f59e0b]"
```